tools/pre-commit.sh

```diff
@@ -12,6 +11,17 @@ then
     exit $need_format
 fi
 
+python -m tools.format_json --check > /dev/null 2>&1
+need_format=$?
+if [ $need_format -ne 0 ]
+then
+    echo "Some JSON files need to be reformatted. Please run:"
+    echo "python -m tools.format_json"
+    exit $need_format
```

Comment:
Can we have this run `python -m tools.format_json`? The black pre-commit currently runs formatting if needed.